api/src/main/java/net/kyori/adventure/bossbar/BossBar.java

```diff
@@ -251,7 +260,7 @@ default float percent() {
    * @return the flags
    * @since 4.0.0
    */
-  @NonNull Set<Flag> flags();
+  @UnmodifiableView @NonNull Set<Flag> flags();
```

Comment:
Fairly sure the `Set` returned here is mutable?

Reply:
luck changed that as part of this PR. Modifying flags wouldn't have worked in the first place, since the listener wouldn't have been called.

Reply:
Makes sense.